src/script/conversation/MessageRepository.ts

```diff
@@ -704,9 +701,6 @@ export class MessageRepository {
     const payload = enableEphemeral && messageTimer ? MessageBuilder.wrapInEphemeral(message, messageTimer) : message;
 
     const injectOptimisticEvent = async () => {
-      if (playPingAudio) {
-        amplify.publish(WebAppEvents.AUDIO.PLAY, AudioType.OUTGOING_PING);
-      }
```

Comment:
This is the main change. There was no reason for the ping sound playing to happen in this function. 
This function is supposed to be general purpose. 

Playing the sound has been moved to a much better place: the `sendPing` method :)

Reply:
Yeah, `sendPing` sounds like a perfect place to publish it 👌 😅